tools/internal_ci/linux/grpc_portability_build_only.cfg

```diff
@@ -26,5 +26,5 @@ action {
 
 env_vars {
   key: "RUN_TESTS_FLAGS"
-  value: "-f portability linux --inner_jobs 8 -j 4 --internal_ci --build_only"
+  value: "-f portability linux c++ --inner_jobs 8 -j 4 --internal_ci --build_only --extra_args -r xds_end2end_test"
```

Comment:
if you're running `--build_only`, the `-r xds_end2end_test` extra args doesn't have any effect AFAICT  (I don't think run_tests.py is clever enough to selectively build tests, it just builds all the tests and then only runs the ones you selected).
If that's the case, let's remove the `--extra_args` since they are making things more confusing.

Reply:
I think having a comment that explains why we only build C++ and why we are passing `-r xds_end2end_test` (if that actually helps) would be useful. Without further explanation, the args being used seem quite arbitrary.